From d45dae1a3a3f2c74d82d887c128159b183ce9bd1 Mon Sep 17 00:00:00 2001
From: Rocky Hao <rocky.hao@rock-chips.com>
Date: Tue, 16 May 2017 10:18:57 +0800
Subject: [PATCH] arm: dts: rk322x-android: update shut mode and enable this
 module

gpio is not connected by default and we suggest cru mode as the default
shut mode.

Change-Id: I74593092b145e51e5f5b52ab028e650b7fe67f5e
Signed-off-by: Rocky Hao <rocky.hao@rock-chips.com>
---
 arch/arm/boot/dts/rk322x-android.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/rk322x-android.dtsi b/arch/arm/boot/dts/rk322x-android.dtsi
index 4ed05d138cf9..0989f623f5eb 100644
--- a/arch/arm/boot/dts/rk322x-android.dtsi
+++ b/arch/arm/boot/dts/rk322x-android.dtsi
@@ -120,6 +120,11 @@
 	};
 };
 
+&tsadc {
+	rockchip,hw-tshut-mode = <0>; /* tshut mode 0:CRU 1:GPIO */
+	status = "okay";
+};
+
 &u2phy0 {
 	status = "okay";
 
-- 
2.35.3

